daedalus/src/Daedalus/BackendApi.purs

```diff
@@ -122,6 +122,12 @@ getProfile = getR ["profile"]
 
 updateProfile :: forall eff. CProfile -> Aff (ajax :: AJAX | eff) CProfile
 updateProfile = postRBody ["profile"]
+
```

Comment:
we don't need `getLocale` and `updateLocale` in BackendApi.purs - we can use getProfile and UpdateProfile which does the same